x-pack/plugins/screenshotting/public/app/app.scss

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+.expression {
```

Comment:
These class names are quite generic which increases the likelihood of style overrides elsewhere in Kibana should this file ever get loaded more globally. To minimize this possibility, we recommend use a unique, 3-letter prefix such as `.scrExpression` and the child could be `.scrExpression__viz`, for example

Reply:
@ryankeairns That's a good point. I followed your recommendation and prefixed that, although I didn't add the prefix to the `.visualization` class since we override styles of the generic class here.
Could you please take another look?